redesign-project/src/index.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: layouts/default.njk
+title: Home
+---
+
+<link rel="stylesheet" href="https://use.typekit.net/hnv1vei.css">
+
+<main class="primary_content">
+    <img src="/images/EmmasMobileBG.png" alt=" " class="background">
+
+    <img src="/images/EmmasDesktopBG.png" alt=" " class="desktop_background">
+
+    <div class="rectangle_large"></div>
+
+
+    <div class="secondary_content">
+        <h2 class="h2__events">Upcoming Events</h2>
+
+        <ul class="ul__events">
+            <li><b><u>Our One-Year Anniversary Party!</u></b><br>Nov 24, 2018<br>8:00 AM – 4:00 PM</li><br>
```

Comment:
Couple of tweaks here:
1. If you use a heading tag on the event title, you can style it based on that class vs relying on the `b` and `u` tags. While they give you the output you want -- bold and underlined text, you can read more about the semantic purposes of these tags at MDN (e.g., [u](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u#Usage_notes) is meant to imply an annotation, not just the presentational effect of "underline."  
2. You can avoid the `br` tags by using the heading tag (which is block level), some spans with `display:block;` or a utility class, and the final `br` can be avoided by applying a margin to the `li`s in this list.
```suggestion
             <li><h3 class="event__title">Our One-Year Anniversary Party!</h3><span class="u-block">Nov 24, 2018</span><span class="u-block">8:00 AM – 4:00 PM</span></li>
```
While the `br` tag does what you want here -- which is create a line break -- it should only be used when the line break is required, like in recreating poems or song lyrics. While the line break here is desired (and okay!), it's not required in the same way -- so the `br` tag is inappropriate.